test/e2e/predictor/test_canary.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 2019 kubeflow.org.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+from kubernetes import client
+
+from kfserving import KFServingClient
+from kfserving import constants
+from kfserving import V1alpha2EndpointSpec
+from kfserving import V1alpha2PredictorSpec
+from kfserving import V1alpha2TensorflowSpec
+from kfserving import V1alpha2InferenceServiceSpec
+from kfserving import V1alpha2InferenceService
+from kubernetes.client import V1ResourceRequirements
+
+from ..common.utils import predict
+from ..common.utils import KFSERVING_TEST_NAMESPACE
+
+api_version = constants.KFSERVING_GROUP + '/' + constants.KFSERVING_VERSION
```

Comment:
Is this API version not already in constants? If it's not, can you add it?

Reply:
Yes, already in https://github.com/kubeflow/kfserving/blob/master/python/kfserving/kfserving/constants/constants.py#L21

Reply:
What i mean is, the constant file should already have this GROUP+VERSION concatenated string so you don't have to repeat this logic. Do you mind creating that as a new constant called constants.KFSERVING_API_VERSION so all tests can consume this?

Reply:
The constant file do not have  GROUP+VERSION concatenated string yet (currently they are two different separate strings) . Will add constants.KFSERVING_API_VERSION. Sorry for misunderstanding your previous comment :-)